invoices/slack.py

```diff
@@ -84,16 +87,35 @@ def refresh_slack_users():
 
 def refresh_slack_channels():
     slack_channels = slack.channels.list().body["channels"]
+    private_channels = slack.groups.list().body["groups"]
     for channel in slack_channels:
-        channel_id = channel.get("id")
-        channel_name = channel.get("name")
-        if channel.get("is_archived"):
-            continue
+        refresh_slack_channels_helper(channel)
+    for private_channel in private_channels:
+        refresh_slack_channels_helper(private_channel)
+
+
+def refresh_slack_channels_helper(channel):
+    channel_id = channel.get("id")
+    channel_name = channel.get("name")
+    if not channel.get("is_archived"):
         SlackChannel.objects.update_or_create(channel_id=channel_id, defaults={
             "name": channel_name,
         })
 
+
 def send_slack_notification(project):
     message = """<!channel> Hi! New project was added: <https://app.10000ft.com/viewproject?id=%s|%s - %s> (created at %s)""" % (project.project_id, project.client, project.name, project.created_at)
     for channel in SlackChannel.objects.filter(new_project_notification=True):
         slack.chat.post_message(channel.channel_id, message)
+
+
+def send_unchecked_weekly_report_notification(year, week):
+
+    for weekly_report in WeeklyReport.objects.all().filter(project_m__enable_weekly_notifications=True, year=year, week=week-1):
+        message = """You have an unapproved weekly report at <https://solinor-finance.herokuapp.com/weekly_report/%s|Solinor Finance service>.""" % weekly_report.weekly_report_id
```

Comment:
This is actually used in line 119 (post_message), so there is no need to create it here